src/components/AddProfile.js

```diff
@@ -187,15 +188,20 @@ class AddProfile extends React.Component {
                           label="Yes"
                           type="radio"
                           name={`isReferral`}
-                          onChange={e => this.setState({ referred: true })}
+                          onChange={e => {this.setState({referred: true});
```

Comment:
you can combine all setState together :
this.setState({referred:true, disabledRefferedBy:false})

Reply:
updated